src/patient-registration/patient-registration.component.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useEffect, useState } from 'react';
+import { Patient } from './patient-registration-helper';
+import { getCurrentUserLocation, getUniquePatientIdentifier, savePatient } from './patient-registration.resource';
+import { createErrorHandler } from '@openmrs/esm-error-handling';
+import { Name } from './field/name/name.component';
+import { Gender } from './field/gender/gender.component';
+import { Birthdate } from './field/birthdate/birthdate.component';
+import { Address } from './field/address/address.component';
+import styles from './patient-registration.css';
+
+const IDENTIFIER_TYPE: string = '05a29f94-c0ed-11e2-94be-8c13b969e334';
+
+interface PatientRegistrationProps {}
+
+export function PatientRegistration(props: PatientRegistrationProps) {
+  const [unknown, setUnknown] = useState<boolean>(false);
+  const [identifier, setIdentifier] = useState<Patient['identifiers'][0]['identifier']>('');
```

Comment:
The `Patient['identifiers'][0]['identifier']` is an indicator that some type should be extracted for simplified use.